src/command_modules/azure-cli-servicefabric/setup.py

```diff
@@ -16,7 +16,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
     cmdclass = {}
 
-VERSION = "0.0.6"
+VERSION = "0.0.7"
```

Comment:
You can leave the version as 0.0.6 the current published version is 0.0.5 so 0.0.6 is okay.
https://pypi.python.org/pypi/azure-cli-servicefabric

Reply:
Cool, rolled back